src/ui/view/dataverse/component/itemOverview/itemOverview.scss

```diff
@@ -0,0 +1,31 @@
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-dataverse-item-overview-type {
+  @include with-theme() {
+    @include noto-sans(700);
+    letter-spacing: 2px;
+    font-size: 14px;
+    color: themed(primary-color-variant-4);
```

Comment:
The color should be different depending on the item, see the `renderItemTypeColor` function in the `dataverseItemCard`